packages/core/src/awsService/cloudWatchLogs/commands/tailLogGroup.ts

```diff
@@ -25,12 +25,16 @@ export async function tailLogGroup(
     if (!wizardResponse) {
         throw new CancellationError('user')
     }
-
+    const awsCredentials = await globals.awsContext.getCredentials()
+    if (awsCredentials === undefined) {
+        throw new ToolkitError('Failed to start LiveTail session: credentials are undefined.')
+    }
     const liveTailSessionConfig: LiveTailSessionConfiguration = {
         logGroupArn: wizardResponse.regionLogGroupSubmenuResponse.data,
         logStreamFilter: wizardResponse.logStreamFilter,
         logEventFilterPattern: wizardResponse.filterPattern,
         region: wizardResponse.regionLogGroupSubmenuResponse.region,
+        awsCredentials: awsCredentials,
```

Comment:
> `lobals.awsContext.getCredentials()` is `async`, which cannot be `await`ed in the constructor since constructors cannot also be aysnc.

We usually add a `create()` "factory function" to work around that, and make the constructor private.